File: src/API/src/ERP.Identity/Model/ViewModels/UserListItemViewModel.cs
namespace ERP.Identity.Model.ViewModels
{
    public class UserListItemViewModel
    {
        public Guid Id { get; set; }
        public string FullName { get; set; } = string.Empty;
        public string Username { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public List<string> Roles { get; set; } = new();
        public List<Guid> RoleIds { get; set; } = new();
        public bool IsActive { get; set; }
    }
}

File: src/ClientApp/src/app/pages/users/view-models/user-list-item.view-model.ts
export interface UserListItemViewModel {
    id: string;
    fullName: string;
    firstName: string;
    lastName: string;
    username: string;
    email: string;
    roles: string[];
    roleIds: string[];
    rolesDisplay: string;
    phoneNumber?: string;
    departmentId?: string;
    managerId?: string;
    isActive: boolean;
}
